Mathlib/Analysis/SpecialFunctions/Log/Basic.lean

```diff
@@ -55,6 +55,13 @@ theorem exp_log (hx : 0 < x) : exp (log x) = x := by
   rw [exp_log_eq_abs hx.ne']
   exact abs_of_pos hx
 
+theorem exp_one_mul_le_exp {x : ℝ} : Real.exp 1 * x ≤ x.exp := by
+  by_cases hx0 : x ≤ 0
+  · apply le_trans
+        (mul_nonpos_of_nonneg_of_nonpos (Real.exp_pos 1).le hx0) (Real.exp_nonneg x)
+  · have h := Real.add_one_le_exp (Real.log x)
+    rwa [← Real.exp_le_exp, Real.exp_add, Real.exp_log (lt_of_not_le hx0), mul_comm] at h
```

Comment:
Can you move that lemma down, below `log_exp`? Right now it is in the middle of lemmas describing the composition of exp and log.
Various code remarks:
- this is placed in the `Real` namespace, so you don't need all the `Real.`
- let's not use dot notation for `exp x`, as `x.exp` is less readable.
- using an explicit argument might make the lemma more usable (in `have` statements notably)
```suggestion
theorem exp_one_mul_le_exp (x : ℝ) : exp 1 * x ≤ exp x := by
  by_cases hx0 : x ≤ 0
  · apply le_trans mul_nonpos_of_nonneg_of_nonpos (exp_pos 1).le hx0) (exp_nonneg x)
  · have h := add_one_le_exp (log x)
    rwa [← exp_le_exp, exp_add, exp_log (lt_of_not_le hx0), mul_comm] at h
```